### DISCUSSION TITLE
Jump to a certain waypoint in the route and then continue routing?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#592] [Q&A] [answered]
Hi, I wonders if it's possible once I load a route with several waypoints and/or shaping points to jump directly to a point for the next target. I know that there's the possibility to overjump the next point but lets say the route starts at point 1 but I'm right now shortly before point10.

Is there a possibility like saying/clicking: Waypoint 10 is my next waypoint please navigate there (don't care about the route before)  and then go ahead and follow the route.?

**Comment by devemux86** [comment]
> route starts at point 1 but I'm right now shortly before point10

Cruiser can start navigation anywhere on the route.

You can also exit the route at any point and enter it later at any point.
(with rerouting off)

**Reply by DIrgi** [reply]
Thank devemux, the magic key "reroute off" ...well..can you tell me where I can find the option is it named exactly rerouting?
Sorry I cannot find it.

And...what will happen let's say again I jump into the route again close to point 10 and rerouting is off. Do I just have to "touch" the route and then cruiser see my location and continue routing or does cruiser automatically guide me to the closest and nearest route contact and then go ahead with routing?

**Reply by devemux86** [reply]
- Rerouting can be turned on / off in "Settings | Navigation | Recalculate route".
- Or long press the direction icon at top-left during navigation to change options.

In navigation settings you can also adjust the distance that is considered off route.
- When you are less than this distance from the route, you can start / resume navigation.
- When you are more than this distance from the route, the off route warning appears.
An automatic rerouting can happen if you have enabled it.

**Reply by DIrgi** [reply, marked as answer]
Cool, thank you for the support
